test.c

```diff
@@ -7,41 +18,73 @@
 
 #include "l3.h"
 
+// Useful constants
+#define L3_MILLION      (1000 * 1000)
+#define L3_NS_IN_SEC    (1000 * 1000 * 1000)
```

Comment:
Add mnemonics to recognize eye-blurring mind-numbing constants with series of `0`s.